tests/test_bench.py

```
@@ -0,0 +1,618 @@
+    Catches the case where a long-form prompt hits --max-tokens and gets cut
+    mid-sentence — the report shows a ⚠ TRUNCATED badge so the reader knows
+    the comparison is unfair."""
+
+    def _build(self, vanilla_out: int, mcp_out: int, cap: int):
+        from evals.runners._providers import PRICING
+        from evals.runners.run_mcp_vs_vanilla import (
+            BenchmarkResult, QueryRun, TrialResult, build_template_context,
+        )
+        from evals.judges.pairwise_judge import aggregate_with_swap
+
+        zero_usage = lambda out: {"input_tokens": 100, "output_tokens": out, "cache_creation_input_tokens": 0, "cache_read_input_tokens": 0}
```

Ruff (0.15.14)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `zero_usage` as a `def`

(E731)